WordPress/Classes/ViewRelated/Settings/ActivityLogDetailViewController.m

```diff
@@ -53,45 +52,14 @@ - (void)viewWillDisappear:(BOOL)animated
 {
     [super viewWillDisappear:animated];
 
-    if (self.popover) {
-        [self.popover dismissPopoverAnimated:animated];
-        self.popover = nil;
-    }
-}
-
-- (void)didReceiveMemoryWarning
-{
-    [super didReceiveMemoryWarning];
-    // Dispose of any resources that can be recreated.
 }
 
 - (void)showShareOptions:(id)sender
 {
-    if (NSClassFromString(@"UIActivityViewController") != nil) {
-        // If UIActivityViewController is available, use it (iOS 6+)
-        UIActivityViewController *activityViewController = [[UIActivityViewController alloc] initWithActivityItems:@[self.logText]
-                                                                                             applicationActivities:nil];
-        if (UI_USER_INTERFACE_IDIOM() == UIUserInterfaceIdiomPad) {
-            if (self.popover && self.popover.isPopoverVisible) {
-                [self.popover dismissPopoverAnimated:YES];
-                self.popover = nil;
-            } else {
-                self.popover = [[UIPopoverController alloc] initWithContentViewController:activityViewController];
-                [self.popover presentPopoverFromBarButtonItem:sender permittedArrowDirections:UIPopoverArrowDirectionAny animated:YES];
-            }
-        } else {
-            [self presentViewController:activityViewController animated:YES completion:nil];
-        }
-    } else {
-        // Otherwise, flip back to an action sheet for < iOS 6
-        UIActionSheet *actionSheet = [[UIActionSheet alloc] initWithTitle:NSLocalizedString(@"Share", @"")
-                                                                 delegate:self
-                                                        cancelButtonTitle:NSLocalizedString(@"Cancel", @"")
-                                                   destructiveButtonTitle:nil
-                                                        otherButtonTitles:NSLocalizedString(@"Mail", @""), nil];
-
-        [actionSheet showFromBarButtonItem:sender animated:YES];
-    }
-
+    UIActivityViewController *activityViewController = [[UIActivityViewController alloc] initWithActivityItems:@[self.logText]
+                                                                                         applicationActivities:nil];
+    activityViewController.modalPresentationStyle = UIModalPresentationPopover;
+    activityViewController.popoverPresentationController.barButtonItem = sender;
+    [self presentViewController:activityViewController animated:YES completion:nil];
```

Comment:
This was (probably) already there. But... the popover is misaligned with regards of the navBar button.

[See screenshot here](https://cloudup.com/cEvI6_DFwuV)